tests/test_stories.py

```diff
@@ -9,57 +9,141 @@
+        self.assertEqual(story['story_name'], story_with_pois['story_name'])
+
+    def test4_get_stories(self):
+        # GET with poi_id parameter
+        story_id_2 = int(os.environ.get('STORY_ID_2'))
+        poi_id_1 = int(os.environ.get('POI_ID_1'))
+        poi_id_2 = int(os.environ.get('POI_ID_2'))
+        r = requests.get('http://127.0.0.1:5000/stories?poi_id={}'.format(poi_id_1))
+        response = r.json()
+        self.assertEqual(response['code'], 200)
+        story = next((s for s in response['result']['stories'] if s['_id'] == story_id_2), None)
+        self.assertEqual(story['story_name'], story_with_pois['story_name'])
+        r = requests.get('http://127.0.0.1:5000/stories?poi_id={}'.format(poi_id_2))
+        response = r.json()
+        self.assertEqual(response['code'], 200)
+        story = next((s for s in response['result']['stories'] if s['_id'] == story_id_2), None)
+        self.assertEqual(story['story_name'], story_with_pois['story_name'])
+
+    def test5_put_stories(self):
+        # PUT with new story name
+        story_id_1 = int(os.environ.get('STORY_ID_1'))
+        r = requests.put('http://127.0.0.1:5000/stories/{}'.format(story_id_1), json=story_edit)
+        response_put = r.json()
+        self.assertEqual(response_put['code'], 200)
+        r = requests.get('http://127.0.0.1:5000/stories')
+        response_get = r.json()
+        story = next((s for s in response_get['result']['stories'] if s['_id'] == story_id_1), None)
+        self.assertEqual(story['story_name'], story_edit['story_name'])
+
+    def test6_put_stories(self):
+        # PUT with blank payload
+        story_id_1 = int(os.environ.get('STORY_ID_1'))
+        r = requests.put('http://127.0.0.1:5000/stories/{}'.format(story_id_1), json=story_empty)
+        response_put = r.json()
+        self.assertEqual(response_put['code'], 200)
+        r = requests.get('http://127.0.0.1:5000/stories')
+        response_get = r.json()
+        story = next((s for s in response_get['result']['stories'] if s['_id'] == story_id_1), None)
+        self.assertEqual(story['story_name'], story_edit['story_name'])
+
+    def test7_delete_stories(self):
+        story_id_1 = int(os.environ.get('STORY_ID_1'))
+        story_id_2 = int(os.environ.get('STORY_ID_2'))
+        r = requests.delete('http://127.0.0.1:5000/stories/{}'.format(story_id_1))
+        response_put = r.json()
+        self.assertEqual(response_put['code'], 200)
+        r = requests.delete('http://127.0.0.1:5000/stories/{}'.format(story_id_2))
+        response_put = r.json()
+        self.assertEqual(response_put['code'], 200)
+        r = requests.get('http://127.0.0.1:5000/stories')
+        response = r.json()
+        story = next((s for s in response['result']['stories'] if s['_id'] == story_id_1), None)
+        self.assertEqual(story, None)
+        story = next((s for s in response['result']['stories'] if s['_id'] == story_id_2), None)
+        self.assertEqual(story, None)

```

Comment:
might also be good to have a test that tries to delete a nonexistent story, which checks if it returns a 404